src/nidcpower/metadata/functions_addon.py

```diff
@@ -52,7 +52,10 @@
     'ConfigureDigitalEdgeSourceTrigger':            { 'parameters': { 2: { 'enum': 'DigitalEdge',                 }, }, },
     'ConfigureDigitalEdgeStartTrigger':             { 'parameters': { 2: { 'enum': 'DigitalEdge',                 }, }, },
     'SendSoftwareEdgeTrigger':                      { 'parameters': { 1: { 'enum': 'SendSoftwareEdgeTriggerType', }, }, },
-    'WaitForEvent':                                 { 'parameters': { 2: { 'enum': 'Event', }, }, },
+    'WaitForEvent':                                 { 'parameters': { 2: { 'enum': 'Event',                       }, }, },
+    'Measure':                                      { 'parameters': { 2: { 'enum': 'MeasurementTypes',            }, }, },
+    'QueryOutputState':                             { 'parameters': { 2: { 'enum': 'OutputStates',                }, }, },
+    'ExportSignal':                                 { 'parameters': { 1: { 'enum': 'Signals',                     }, }, },
```

Comment:
Do we have system tests for these to add to this review?

Reply:
Tests are being created. Will submit in another PR.